airmozilla/main/templates/main/main_base.html

```diff
@@ -148,6 +149,7 @@
   <script src="{{ static('js/libs/include.js') }}"></script>
   <script src="{{ static('browserid/api.js') }}"></script>
   <script src="{{ static('browserid/browserid.js') }}"></script>
+  <script src="{{ static('starred/js/star_event.js') }}"></script>
```

Comment:
It might sound unimportant but if you put this *above* `js/libs/include.js` this code will execute slightly ahead of the other stuff related to persona. 
We want that stuff to take effect really quickly so that the user (a user with "quick eyes") doesn't notice that her starred events start out as a hollow star and then suddenly changes to a filled star.